addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -135,6 +159,25 @@ const KnowledgeArticleFormRenderer = FormRenderer.extend(KnowledgeTreePanelMixin
         });
     },
 
+    /**
+     * Update the chatter.
+     * Called by the @see KnowledgeController when a reload is initiated by the
+     * chatter.
+     */
+    async updateChatter() {
```

Comment:
Why is updateChatter public and _renderChatter private ?

Reply:
Because updateChatter is a method intended to be used by the controller, and _renderChatter is a method called when the chatter is rendered (when the user clicks on the chatter button). The chatter is not rendered initially with the view (since it is hidden => perf++), only when it is needed